pkg/rpc/adminservice/named_entity.go

```diff
@@ -0,0 +1,78 @@
+package adminservice
+
+import (
+	"context"
+
+	"github.com/lyft/flyteadmin/pkg/rpc/adminservice/util"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (m *AdminService) GetNamedEntity(ctx context.Context, request *admin.NamedEntityGetRequest) (*admin.NamedEntity, error) {
+	defer m.interceptPanic(ctx, request)
```

Comment:
this shouldn't be necessary, we start the admin server with a panic recovery wrapper: https://github.com/lyft/flyteadmin/blob/master/pkg/rpc/adminservice/base.go#L60

Reply:
I can remove it. I was following the pattern in other RPC handlers.

Reply:
oops, disregard then!